### ISSUE TITLE
Cannot find name 'ReactComponentType' when compiling with TypeScript 3.7.5

### ISSUE THREAD

**Issue by thehellbean** [software-mansion/react-native-gesture-handler#986]
Behavior:
```
$ node_modules/.bin/tsc
node_modules/react-native-gesture-handler/react-native-gesture-handler.d.ts:457:40 - error TS2304: Cannot find name 'ReactComponentType'.

457   export const GestureHandlerRootView: ReactComponentType<ViewProps>;
```

Expected behavior:
I expected it to pass typecheck when compiling with typescript

**Comment by h4nyu** [comment]
https://github.com/software-mansion/react-native-gesture-handler/pull/973

**Comment by parinha** [comment]
tsconfig.json
```
{
  "compilerOptions": {
    "skipLibCheck": true ,
    ....
  }
}
```

**Comment by jakub-gonet** [comment]
Should be resolved by #945
